src/main/java/org/broadinstitute/hellbender/tools/dragstr/EstimateDragstrParameters.java

```diff
@@ -0,0 +1,935 @@
+        }
+    }
+
+    private Optional<String> resolveSampleName(List<String> sampleNames) {
+        if (sampleNames.size() > 1) {
+            throw new GATKException("the input alignment(s) have more than one sample: " + String.join(", ", sampleNames));
+        } else if (sampleNames.isEmpty() || sampleNames.get(0) == null) {
+            logger.warn("there is no sample id in the alignment header, assuming that all reads and read/groups make reference to the same anonymous sample");
+            return Optional.empty();
+        } else {
+            return Optional.of(sampleNames.get(0));
+        }
+    }
+
+    private void checkSequenceDictionaryCompatibility(final SAMSequenceDictionary reference, final SAMSequenceDictionary strTable) {
+        final SequenceDictionaryUtils.SequenceDictionaryCompatibility compatibility = SequenceDictionaryUtils.compareDictionaries(reference, strTable, false);
+        switch (compatibility) {
+            case IDENTICAL: return;
+            case SUPERSET: return;
+            // probably these two below aren't ever be returned since we ask for no check on order but
+            // adding them it just in case
+            case NON_CANONICAL_HUMAN_ORDER: return; // we don't care about the order.
+            case OUT_OF_ORDER: return; // we don't care about the order.
+            default:
+                throw new GATKException("the reference and str-table sequence dictionary are incompatible: " + compatibility);
+        }
+    }
+
+    private PrintWriter openSitesOutputWriter(final String sitesOutput) {
+        return sitesOutput == null ? new PrintWriter(new NullOutputStream())
+                : new PrintWriter(BucketUtils.createFile(sitesOutput));
+    }
+
+    private void outputDownSampledSiteDetails(final StratifiedDragstrLocusCases finalSites,
+                                              final PrintWriter writer,
+                                              final int minDepth,
+                                              final int samplingMinMQ,
+                                              final int maxSup) {
+        if (sitesOutput != null) {
+            for (final DragstrLocusCases[] periodCases : finalSites.perPeriodAndRepeat) {
+                for (final DragstrLocusCases repeatCases : periodCases) {
+                    for (final DragstrLocusCase caze : repeatCases) {
+                        outputSiteDetails(writer, caze, caze.qualifies(minDepth, samplingMinMQ, maxSup) ? "used" : "skipped");
+                    }
+                }
+            }
+        }
+    }
+
+    /**
+     * Holds the minimum counts for each period, repeat-length combo.
+     * If there is lack of data for any of these we use the default param
+     * tables. Missing values, row (periods) or columns (repeat-length) are
+     * interpreted as 0.
+     */
+    private static final int[][] MINIMUM_CASES_BY_PERIOD_AND_LENGTH =
+            // @formatter:off ; prevents code reformatting by IntelliJ
```

Comment:
I suggest to leave this one here.... as it the way to prevent code reformatting by IntelliJ, the most common (perhaps the only IDE, use by the GATK team. That format makes reading table literal enclosed more straight forward.